### PR TITLE
Added new test for bsc#947398

### PR DESCRIPTION
**Pull request by kobliha**
- Leap was not supported for upgrade
- This new test enhances the test coverage and also tests new entry to control file which can be found here at https://github.com/yast/skelcd-control-openSUSE/pull/37

### CODE REVIEW

File: test/update_test.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@
 Yast.import "Installation"
 Yast.import "ProductControl"
 Yast.import "ProductFeatures"
+Yast.import "ProductFeatures"
```
Comment (imobach) [inline test/update_test.rb]: Duplicated?

**Comment by aschnell** [comment]
LGTM

File: test/update_test.rb (hunk 1 above)
Comment (kobliha) [inline test/update_test.rb]: You are right, I was "fixing" a problem that did not exist

**Comment by kobliha** [comment]
@aschnell @imobach thx for review! :)

**Pull request merged**